src/test/java/edu/kis/vh/nursery/DefaultCountingOutRhymerTest.java

```diff
@@ -40,16 +41,16 @@ public void testIsFull() {
 		}
 
 		boolean result = rhymer.isFull();
-		Assert.assertEquals(true, result);
+		Assert.assertEquals(false, result);

```

Comment:
Is it a correct approach to fixing errors showed by tests? 
If STACK_CAPACITY is 12 then 12 elements should be added to the stack and it should be full. Wrong fix.